samples/nrf9160/lwm2m_client/src/lwm2m/include/lwm2m_app_utils.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <net/lwm2m.h>
+#include <drivers/sensor.h>
+
+#ifndef LWM2M_APP_UTILS_H__
+#define LWM2M_APP_UTILS_H__
+
+/* LwM2M IPSO Object IDs */
+#define IPSO_OBJECT_SECURITY_ID 0
```

Comment:
Security, Server and Devcie are not IPSO objects but regular LwM2M objects, the names are misleading.

Reply:
Changed to LWM2M_OBJECT_..._ID